test/sandbox/app/components/slots_v2_component.html.erb

```diff
@@ -1,12 +1,16 @@
 <div class="card <%= @classes %>">
+  <% if title? %>
```

Comment:
Should I create a separate sandbox component to test this behavior, instead of altering the existing one?

Reply:
I think we have other tests that cover getting slots, so I'm fine with this change 👍